docs/source/use_cases/analyze_reads/calculate_coverage.rst

```diff
@@ -45,6 +48,13 @@ write access, you may use it.  If not, you can do the following to create one:
 The ``id`` that is generated in the response output is the output dataset id you should use when running
 the pipeline.
 
+Download ALPN
+-------------
+
+Running this pipeline locally requires that you have the ALPN jar that matches your JRE version
+on your computer.  See the `ALPN documentation <http://www.eclipse.org/jetty/documentation/9.2.10.v20150310/alpn-chapter.html>`_ for a table of which ALPN jar to use.
+
+You will not need to provide the ALPN flag or jar if you run the pipeline on Google cloud instead of locally.
```

Comment:
That is correct, you do not have to set it, but dataflow checks if that flag exists and is true in order to know whether or not to include ALPN in the class path, so the flag has to be in options in order to make this work.  There is a bug that requires you to have to call options.useAlpn() if you are depending on the default, otherwise it does not get registered into the pipeline.

I believe that the doc is internal and thus I unfortunately cannot share it with you.